src/utils/drag-dom-element.js

```diff
@@ -0,0 +1,45 @@
+function dragDomElement(el, selector) {
+    if (!el) return;
```

Comment:
this feels like something we should use jquery.draggable for instead of reinventing the wheel

Reply:
jquery-ui is not compatible with browserify. see https://jqueryui.com/upgrade-guide/1.12/#official-package-on-npm

Reply:
I'm trying https://www.npmjs.com/package/draggable